spec/features/admin/order_cycles/simple_spec.rb

```diff
@@ -604,6 +604,17 @@
     end
   end
 
+  scenario "modify the minute of a order cycle with the keyboard, check that the modifications are taken into account" do
+    order_cycle = create(:simple_order_cycle, name: "Translusent Berries")
+    login_as_admin_and_visit admin_order_cycles_path
+    expect(page).to have_selector "tr.order-cycle-#{order_cycle.id}"
+    find("#oc#{order_cycle.id}_orders_close_at").click
+    datetime = Time.at(Time.zone.local(2040, 10, 17, 0o6, 0o0, 0o0))
+    input = find(".flatpickr-calendar.open .flatpickr-minute")
+    input.send_keys datetime.strftime("%M").to_s.strip
+    expect(page).to have_content "You have unsaved changes"
+  end
```

Comment:
this might be my personal preference but I prefer to split this big block. I tend to keep the various `expect` calls separate together with their own setup.

Reply:
Actually, this line:
```ruby
expect(page).to have_selector "tr.order-cycle-#{order_cycle.id}"
```
is a bit superfluous (as it is already tested elsewhere). 
Thus, this block as only one `expect()`. What do you think then?

Reply:
that's even better.